anonymize/anonymize.go

```diff
@@ -16,36 +13,34 @@ var tokenSep = regexp.MustCompile(`\s|[,;]`)
 var userSep = regexp.MustCompile("[._-]")
 var adjacentSecrets = regexp.MustCompile(fmt.Sprintf(`%s(\s%s)+`, anonym, anonym))
 
+const namesSep = `(\b|[!,.?:-]+)`
+
+var namesRe = regexp.MustCompile(
+	fmt.Sprintf(
+		`(?i)%s%s%s`,
+		namesSep,
+		strings.Join(
+			names,
```

Comment:
Does `names` have to be hardcoded?

Reply:
Good question. We'll see how many requests from customers we get to add names for anonymization. I hope it's not gonna be too often. Otherwise we'll use recordlists (could be edited with items added / removed via admin cp).

Reply:
I suggest the names list ought to be maintained by the client side and not holster.

Reply:
How about making `names` visible `Names` so client can redefine it? Plus a comment in Anonymize docstring. I'd like the function to work out of the box with some default. I was thinking about passing names as a variadic variable, so that client can redefine it if they'd like but `Anonymize()` already have variadic strings (secrets).